langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -425,6 +616,14 @@ public isolated function toFixedString(float x, int? fractionDigits) returns str
 # to represent the result. If `x` is zero, the exponent is zero. A zero exponent
 # is represented with a `+` sign.
 #
+# ```ballerina
+# // Take the string representation of scientific notation without decimal points.
+# string strScientificIntValue = float:toExpString(12.456, 0);
```

Comment:
Is saying int value here (and in the previous example) correct?

Reply:
Okay, let's have one example here then.

Reply:
yes, but here our target is to expose how the function can use. So because of there is no optional arguments, I guess it is better to have one example. wdyt?